index.markdown

```diff
@@ -621,6 +621,11 @@ Capital One credit card that's directly targeted for people with no credit
 history/newcomers. Beware this card is free for the first year, but costs around
 $30 per year after that.
 
+#### Wells Fargo
+
+Stefan got declined twice trying to get a secured credit card there. Both times they 
+said my SSN was invalid. If you want a CC, wouldn't recommend Wells Fargo.
```

Comment:
I applied for a secured credit card and got rejected twice. I should have mentioned it was a secured.